pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -66,8 +66,6 @@
 from .api_wrappers import QuantinuumAPIError, QuantinuumAPI
 
 _DEBUG_HANDLE_PREFIX = "_MACHINE_DEBUG_"
-QUANTINUUM_URL_PREFIX = "https://qapi.quantinuum.com/"
```

Comment:
What happend to the url?

Reply:
This is defined here: https://github.com/CQCL/pytket-quantinuum/blob/develop/pytket/extensions/quantinuum/backends/api_wrappers.py#L77

The variable `QUANTINUUM_URL_PREFIX` is never used.